spec/request/api/v1/auth/reset_password_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+describe 'PUT api/v1/auth/password', type: :request do
+  let(:user) { create(:user, :confirmed) }
+  let(:reset_url) { email_link_url last_email }
+  let(:params) do
+    {
+      email: user.email,
+      redirect_url: 'http://localhost:3000'
+    }
+  end
+  let(:new_user) { build(:user) }
```

Comment:
Why are you building the `user`? You should create the user in order to check that the password has actually changed.